templates/plugin/content/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
Is right? Maybe for now we can just not include any license by default?

Reply:
Yeah, let's remove for now.